src/components/viewContainer.js

```diff
@@ -44,11 +44,11 @@ define(["browser", "dom", "layoutManager", "css!components/viewManager/viewConta
 
 
             if (isPluginpage) {
-                require(["legacy/dashboard"]);
+                require(["legacyDashboard"]);
```

Comment:
could you explain this?

Reply:
I'm just consolidating the legacy script locations in `site.js` so we have a central location for items we need to remove eventually.